src/PersonalMessageManager.ts

```diff
@@ -0,0 +1,332 @@
+import { EventEmitter } from 'events';
+import BaseController, { BaseConfig, BaseState } from './BaseController';
+import { validatePersonalSignMessageData, normalizeMessageData } from './util';
+const random = require('uuid/v1');
```

Comment:
import ?

Reply:
honestly I didn't question this, TransactionController uses the same and imports it in the same way